test/support/helpers/company-contact.helper.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+/**
+ * @module CompanyContactHelper
+ */
+
+const CompanyContactModel = require('../../../app/models/company-contact.model.js')
+const { generateUUID } = require('../../../app/lib/general.lib.js')
+/**
+ * Add a new company contact
+ *
+ * If no `data` is provided, default values will be used. These are
+ *
+ * - `id` - [random UUID]
+ * - `companyId` - [random UUID]
+ * - `contactId` - [random UUID]
+ * - `roleId` - [random UUID]
+ *
+ * @param {Object} [data] Any data you want to use instead of the defaults used here or in the database
+ *
+ * @returns {Promise<module:CompanyContactModel>} The instance of the newly created record
+ */
+function add (data = {}) {
+  const insertData = defaults(data)
+
+  return CompanyContactModel.query()
+    .insert({ ...insertData })
+    .returning('*')
+}
+
+/**
+ * Returns the defaults used
+ *
+ * It will override or append to them any data provided. Mainly used by the `add()` method, we make it available
+ * for use in tests to avoid having to duplicate values.
+ *
+ * @param {Object} [data] Any data you want to use instead of the defaults used here or in the database
+ */
+function defaults (data = {}) {
+  const defaults = {
+    id: generateUUID(),
```

Comment:
You don't need to default the `id:` if the legacy table automatically generates them. There are the odd few that don't hence we might have specified this in some helpers. But they are the exception rather than the norm.

```suggestion
```

N.B. This will also mean it can be dropped from the JSDoc description.